evaluation/scripts/perturb_fairscore.py

```diff
@@ -0,0 +1,122 @@
+import argparse
```

Comment:
I think I'm a bit out of context for the PR, but if you could add comments to the file quickly describing what it does / cmd line usage, we can probably approve and merge

Reply:
This way future contributors also know why we didn't merge this with perturb.py and needed a separate script